client/webserver/site/src/js/markets.js

```diff
@@ -959,6 +959,16 @@ export default class MarketsPage extends BasePage {
   handleBookRoute (note) {
     app.log('book', 'handleBookRoute:', note)
     const mktBook = note.payload
+    const { book } = mktBook
+    // gets first price value from buy or from sell, so we can show it on
+    // title.
+    const firstKnownValue = book.buys[0] ? book.buys[0] : book.sells[0]
```

Comment:
You can update the title after the call to `this.handleBook` below and just use `this.midGap()` for the value.